cmd/snap-confine/ns-support.c

```diff
@@ -928,6 +928,39 @@ void sc_store_ns_info(const sc_invocation * inv)
 	debug("saved mount namespace meta-data to %s", info_path);
 }
 
+void sc_store_ns_fstab(const sc_invocation *inv)
```

Comment:
given that snap-update-ns will write more there, store is a bit unclear in the name. sc_initialize_with_root_ns_fstab? anyway it's also a bit unclear whether this helper belongs here or in mount-support.c, as ultimately is about data shared with snap-update-ns which is invoked from the latter?

Reply:
I moved and renamed to `sc_initialize_ns_fstab()`, I left out `with_root` as it seems a detail.